src/scrapers/visa-cal.ts

```diff
@@ -362,7 +362,7 @@ class VisaCalScraper extends BaseScraperWithBrowser<ScraperSpecificCredentials>
       preAction: this.openLoginPopup,
       postAction: async () => {
         try {
-          await waitUntilElementFound(this.page, 'button.btn-next-time');
+          await waitUntilElementFound(this.page, 'button.butn-small-light-grey.btn-next-time');
```

Comment:
I changed it to `waitForNavigation` and it worked for me, and I have the popup.

```suggestion
          await waitForNavigation(this.page);
```

Reply:
I changed it, @maorsom @shaiu